### PR TITLE
add openssl_error_string() output to every message

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: lib/Pikirasa/Exception.php [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
 namespace Pikirasa;
 
-class Exception extends \Exception { }
+class Exception extends \Exception
+{
+    public function __construct($message, $code = 0, \Exception $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+
+        if ($openSSlErrorMessage = openssl_error_string()) {
```
Comment (vlucas) [inline lib/Pikirasa/Exception.php]: This `$openSSlErrorMessage` variable is unused below. Either drop it, or use it below.

Comment (greg0ire) [inline lib/Pikirasa/Exception.php]: Woops! I knew I forgot something. This is fixed.

**Comment by vlucas** [comment]
Thanks! Looks good :+1:

**Comment by vlucas** [comment]
Tagged a `v1.0.4` release with this fix as well as #6

**Comment by greg0ire** [comment]
Great!

**Pull request merged**